src/components/src/main/java/org/apache/jmeter/extractor/RegexExtractor.java

```diff
@@ -71,6 +72,8 @@ public class RegexExtractor extends AbstractScopedTestElement implements PostPro
 
     private static final String UNDERSCORE = "_";  // $NON-NLS-1$
 
+    private static final String FAILIFNOTFOUND = "RegexExtractor.fail_if_not_found";
```

Comment:
Please declare element properties in https://github.com/apache/jmeter/blob/09c3f810dda7efaf1510ac062454d978d90a0741/src/components/src/main/kotlin/org/apache/jmeter/extractor/RegexExtractorSchema.kt

Reply:
@vlsi I've updated the code for the schema